### PR TITLE
test: added breeze api test cases

### PR DESCRIPTION
**Pull request by YogevBokobza**
## Description

> Getting 100% code coverage by remove unused code and add tests

**Related issue (if any):** fixes #issue_number_goes_here

## Checklist

- [x] I have followed this repository's contributing guidelines.
- [x] I will adhere to the project's code of conduct.

## Additional information

> Anything else?

### CODE REVIEW

File: tests/test_api_tcp_client.py
Comment (TomerFi) [inline tests/test_api_tcp_client.py:315]: It's been a long time since I worked on this project, I don't remember what goes into these packets. Are you there's isn't anything important here?

File: .coveragerc
Comment (TomerFi) [inline .coveragerc]: If this is here for the sole purpose of reaching 100% code coverage, we can drop it.
I'm ok with our current 98%. :-)

**Review by TomerFi — CHANGES_REQUESTED** [review]

File: tests/test_api_tcp_client.py
Comment (YogevBokobza) [reply tests/test_api_tcp_client.py:315]: Yes.
As part of this coverage improvement, I just saw that part of a code didn't reach by any test so I added a test to support that.

In this file, There is also part of the code which is not used anywhere..

File: .coveragerc
Comment (YogevBokobza) [reply .coveragerc]: Why drop it?
Those exclusions make sense and improve coverage and also are sensitive to decreased coverage over-time (As did happen in [#710)](https://github.com/TomerFi/aioswitcher/pull/710)

Comment (TomerFi) [reply .coveragerc]: 1) What's wrong with 98% coverage? Who says 100% is required or even a best practice?
2) If we were too add this, and I don't think we should even bother, we use `pyproject.toml` for coverage configuration.

File: tests/test_api_tcp_client.py
Comment (TomerFi) [reply tests/test_api_tcp_client.py:315]: Yes, I noticed that - thanks.
I was only talking about the packet itself, the one you used in the `assert_that` statement, just making sure it doesn't contain any personal/sensitive data before merging.

Comment (YogevBokobza) [reply tests/test_api_tcp_client.py:315]: No its not. its the same packet as used before in the code so it should be fine :)

File: .coveragerc
Comment (YogevBokobza) [reply .coveragerc]: Well, I don't know then.. I think if it's possible to be at 100% why not.
Those excludes make sense in my opinion as why would you test abstract class? why would you test RuntimeError?

It's your call. 
If you want you can merge this PR if not then no..

Comment (TomerFi) [reply .coveragerc]: Let's remove this file, seems redundant to me.

Comment (YogevBokobza) [reply .coveragerc]: As you wish.. File removed.

**Review by TomerFi — APPROVED** [review]

File: .coveragerc
Comment (TomerFi) [reply .coveragerc]: Thank you very much for your hard work - merging this now.
For future reference, this repository is already full of annoying configuration files, whenever possible, it's prefered using `pyproject.toml'. Coverage is already configured there:

https://github.com/TomerFi/aioswitcher/blob/0544a5047ae1e9ae738f5b909b67c9af793e355e/pyproject.toml#L100

**Comment by TomerFi** [comment]
@allcontributors add @YogevBokobza for test and maintenance

**Pull request merged**
